Gemfile

```diff
@@ -2,3 +2,4 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in rex-socket.gemspec
 gemspec
+gem 'pry', '~> 0.13.1'
```

Comment:
I don't mind this being here, but it'd be great to move this to being a developer dependency instead if possible :+1:

https://github.com/rapid7/rex-text/blob/fce504e9ea8632f1254e74e83031165ccb723014/Gemfile#L6-L8